libs/managers/GitTools.cs

```diff
@@ -3,14 +3,40 @@
 using System;
 using System.Diagnostics;
 using System.IO;
-
+using System.Runtime.InteropServices;
+using Godot;
+using Godot.Collections;
 namespace GitTools
 {
-    public class GitRunner
+    class Util{
+        public static String GetGit()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))

```

Comment:
I assume this is more what we are looking for

Reply:
The way that I determine it in Godot is with Compile Time flags, such as:
#if GODOT_WINDOWS || GODOT_UWP  // Windows
#elif GODOT_LINUXBSD || GODOT_X11 // Linux
#elif GODOT_MACOS || GODOT_OSX // Mac OS
#endif

This way, only the code between the true if statement will be put in, based upon the platform it is being compiled for.

Reply:
Ahh I like that.. silly me. Still new to C# but have used C before.